Password.java

```diff
@@ -0,0 +1,58 @@
+public class Password {
+	private static final String ALGORITHM = "AES";
+	private static final String INIT_VECTOR = "ZwNnZZXfhanqzUtZV6gQQg==";
+	private static final String KEY = "upW8BpO89Qwilpr4XG4j3Q==";
+	private static final String PADDING = "/CBC/PKCS5Padding";
+	private Key key;
+
+	public Key getKey(){
+		return this.key;
+	}
+	
+	public String getINIT_VECTOR(){
```

Comment:
is there a need for getter for a static final field? i think if field is static final and it will be used somewhere else then the fild should be simply public, getter is not needed